inst/extdata/population.csv

```diff
@@ -1,18 +1,18 @@
-"age","uk","england","wales","scotland","northern_ireland","north_west","north_east_and_yorkshire","midlands","east_of_england","london","south_east","south_west"
-"0",3857263,3299637,165542,271715,120369,420939,489309,624065,392286,615881,507392,296855
-"5",4149852,3538206,183784,299316,128546,438916,522687,661208,417933,599855,557684,325583
-"10",3953866,3354246,180819,294674,124127,412623,492140,623535,386327,524126,527782,307586
-"15",3656968,3090232,172381,281958,112397,393595,480725,605870,349227,464768,497816,304574
-"20",4153080,3487863,202402,347456,115359,451620,575863,702473,354798,563241,525111,339548
-"25",4514249,3801409,208260,382255,122325,484286,577743,710047,391210,784361,531094,337183
-"30",4497132,3807954,192954,369463,126761,461260,541019,666796,414426,849074,527497,327525
-"35",4395667,3733642,184920,352522,124583,441192,521626,651730,426002,783331,565692,327910
-"40",4019539,3414297,169556,319432,116254,399928,475129,603274,402791,646347,551341,311774
-"45",4402122,3715812,199072,361458,125780,473672,572987,713583,448631,596141,620052,374176
-"50",4661015,3907461,220480,401090,131984,494502,605129,739225,464715,564205,644651,403024
-"55",4405908,3670651,217480,393123,124654,460456,573996,675572,421265,479470,582461,380109
-"60",3755185,3111835,192853,344693,105804,391498,501802,579975,362595,378068,491505,338931
-"65",3368199,2796740,182142,299444,89873,363640,464318,551557,341872,309586,458710,332910
-"70",3318867,2779326,179286,278856,81399,347142,440635,533581,341711,265847,462533,333351
-"75",2325296,1940686,126785,195951,61874,239787,298198,368463,227741,189753,313538,225646
-"80",3362599,2836964,174163,269894,81578,337891,433619,526725,349658,294027,487502,339312
+"age","uk","england","wales","scotland","northern_ireland","east_of_england","london","midlands","north_east_and_yorkshire","north_west","south_east","south_west"
```

Comment:
Can you update the README in the inst/extdir directory indicating where these numbers have come from, probably leaving a reference to the old version too